tasks/optional-label-check.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: optional-label-check
+spec:
+  params:
+    - name: sourceImage
+      description: the fully qualified image name
+      default: quay.io/cvpops/test-index:v4.10
+
+  steps:
+    - name: inspect-image
```

Comment:
I do not see here reason of splitting this task into two steps. They are using the same image.

Reply:
Good point, merged